src/Component/HItApi.js

```diff
@@ -0,0 +1,43 @@
+import React, { useState, useEffect } from 'react';
+import axios from 'axios';
+
+function DisplayQuote() {
```

Comment:
- [ ] The milestone requires you create a `<Quotes>` component, however you have named yours `<DisplayQuotes>` please kindly change this component accordingly. 👍 
- [ ] Also the name of the file holding this component should be descriptive giving details about what the component does such as `Quotes,js` rather than what you currently have which is `HitApi.js`.